contracts/src/TeleporterTokenSource.sol

```diff
@@ -71,11 +101,69 @@ abstract contract TeleporterTokenSource is
         feeTokenAddress = feeTokenAddress_;
     }
 
+    function _registerDestination(
+        bytes32 destinationBlockchainID,
+        address destinationBridgeAddress,
+        uint256 initialReserveImbalance,
+        uint256 tokenMultiplier,
+        bool multiplyOnSend
+    ) internal {
+        require(
+            destinationBlockchainID != bytes32(0),
+            "TeleporterTokenSource: zero destination blockchain ID"
+        );
+        require(
+            destinationBlockchainID != blockchainID,
+            "TeleporterTokenSource: cannot register bridge on same chain"
+        );
+        require(
+            destinationBridgeAddress != address(0),
+            "TeleporterTokenSource: zero destination bridge address"
+        );
+        require(
+            tokenMultiplier > 0 && tokenMultiplier < 1e18,
```

Comment:
We should probably have max token multiplier set somewhere, instead of being a magic number here

Reply:
added a constant